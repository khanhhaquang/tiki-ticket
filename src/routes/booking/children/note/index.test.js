import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import Note from './index';

describe('Test Booking/Note component', () => {
	test('Snapshot note', () => {
		const component = renderer.create(<Note />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});

	const component = mount(<Note />);
	test('Check note states and types', () => {
		expect(component.find('.states')).toBeTruthy();
		expect(component.find('.ticket-types')).toBeTruthy();
	});

	test('Check info price', () => {
		expect(component.find('.price.total')).not.toBeNaN();
		expect(component.find('.price.currency')).toBeTruthy();
	});
});
