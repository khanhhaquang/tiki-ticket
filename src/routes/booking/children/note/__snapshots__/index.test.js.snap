// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Booking/Note component Snapshot note 1`] = `
<div
  className="sc-bdVaJa eyMFbt note-wrapper"
>
  <section
    className="states"
  >
    <div
      className="sc-bwzfXH jlgJKy item"
    >
      <span
        className="pre booked"
      />
      <span
        className="content"
      >
        Đã đặt
      </span>
    </div>
    <div
      className="sc-bwzfXH jlgJKy item"
    >
      <span
        className="pre selected"
      />
      <span
        className="content"
      >
        Đang chọn
      </span>
    </div>
  </section>
  <section
    className="ticket-types"
  >
    <div
      className="sc-bwzfXH jlgJKy item"
    >
      <span
        className="pre standard"
      />
      <span
        className="content"
      >
        Standard - 60.000đ
      </span>
    </div>
    <div
      className="sc-bwzfXH jlgJKy item"
    >
      <span
        className="pre vip"
      />
      <span
        className="content"
      >
        VIP - 90.000đ
      </span>
    </div>
    <div
      className="sc-bwzfXH jlgJKy item"
    >
      <span
        className="pre deluxe"
      />
      <span
        className="content"
      >
        Deluxe - 110.000đ
      </span>
    </div>
  </section>
</div>
`;
